JoyboyCommunity/src/app/App.tsx

```diff
@@ -18,6 +20,28 @@ export default function App() {
   const tips = useTips();
   const {showToast} = useToast();
 
+  const {showDialog, hideDialog} = useDialog();
+
+  const {chain} = useNetwork();
```

Comment:
Hi @ugur-eren,

I’ve applied the recent changes, but I’m encountering an issue where the hook always returns Sepolia, even when I’m on Mainnet. Is this expected behavior?

I’ve looked through the [documentation](https://book.starknet.io/ch02-08-starknet-react.html) and the [Starknet React documentation](https://starknet-react.com/hooks/usenetwork), and it seems like it should be working. I’m not sure if this is an issue with the library or if something in our setup might be incorrect.

I also tried adding the commented-out code in JoyboyCommunity/src/app/StarknetProvider.tsx for argentMobileConnector, but I’m missing the WALLET_CONNECT_ID. 

Do you have any ideas on what might be causing this issue?

Thanks!

Reply:
Sorry.. my mistake. Seems like useNetwork returns the provided network in the StarknetConfig, instead of the account's currently connected one. I've fixed it myself, sorry for inconvenience

Reply:
Hey no prob 😊